PLUB/Sources/Views/Home/Clipboard/Cell/BoardCollectionViewCell.swift

```diff
@@ -16,6 +16,7 @@ final class BoardCollectionViewCell: UICollectionViewCell {
   static let identifier = "BoardsCollectionViewCell"
   
   // MARK: - UI Components
+  var feedID: Int?
   
```

Comment:
UI Components라기엔 무리가 있습니다.
```suggestion
  //MARK: - Properties
  
  var feedID: Int?
  
  // MARK: - UI Components
  
```

Reply:
아 저 부분 승현님 코드 그대로 긁어오다가 저렇게 포함됬나보네요 변경하도록 하겠습니다 !!